jobs/vsphere_cpi/spec

```diff
@@ -26,6 +26,8 @@ properties:
     description: Datacenters in vCenter to use (value is an array of Hashes representing datacenters and clusters, See director.yml.erb.erb)
   vcenter.enable_auto_anti_affinity_drs_rules:
     description: Creates DRS rule to place VMs on separate hosts.
+  vcenter.upgrade_hw_version:
+    description: Upgrades virtual machines to latest virtual harware version supported on the esx host.
```

Comment:
typos - Upgrades virtual machines to latest virtual hadrware version supported on the ESXi host.